sdk/freestanding-execution-engine/src/main.rs

```diff
@@ -200,57 +209,42 @@ fn parse_command_line() -> Result<CommandLineOptions, Box<dyn Error>> {
     } else {
         return Err("No binary file provided.".into());
     };
-    let data_sources = if let Some(data) = matches.values_of("data") {
-        let data_sources: Vec<String> = data.map(|e| e.to_string()).collect();
+    let input_sources = if let Some(data) = matches.values_of("input") {
+        let input_sources: Vec<String> = data.map(|e| e.to_string()).collect();
         info!(
             "Selected {} data sources as input to computation.",
-            data_sources.len()
+            input_sources.len()
         );
-        data_sources
+        input_sources
     } else {
         Vec::new()
     };
 
-    let dump_stdout = {
-        let dump_stdout = matches
-            .value_of("dump-stdout")
-            .unwrap_or(DEFAULT_DUMP_STDOUT);
-        if let Ok(dump_stdout) = bool::from_str(dump_stdout) {
-            if dump_stdout {
-                info!("stdout configured to be dumped before exiting.");
-            }
-            dump_stdout
-        } else {
-            return Err(format!("Expecting a boolean, but found {}", dump_stdout).into());
-        }
+    let output_sources = if let Some(data) = matches.values_of("output") {
+        let output_sources: Vec<String> = data.map(|e| e.to_string()).collect();
+        info!(
+            "Selected {} data sources as input to computation.",
+            output_sources.len()
+        );
+        output_sources
+    } else {
+        Vec::new()
     };
 
-    let dump_stderr = {
-        let dump_stderr = matches
-            .value_of("dump-stderr")
-            .unwrap_or(DEFAULT_DUMP_STDERR);
-        if let Ok(dump_stderr) = bool::from_str(dump_stderr) {
-            if dump_stderr {
-                info!("stderr configured to be dumped before exiting.");
-            }
-            dump_stderr
-        } else {
-            return Err(format!("Expecting a boolean, but found {}", dump_stderr).into());
-        }
+    let enable_clock = if matches.is_present("enable-clock") {
+        true
+    } else {
+        DEFAULT_ENABLE_CLOCK
     };
-
-    let enable_clock = {
-        let enable_clock = matches
-            .value_of("enable-clock")
-            .unwrap_or(DEFAULT_ENABLE_CLOCK);
-        if let Ok(enable_clock) = bool::from_str(enable_clock) {
-            if enable_clock {
-                info!("Clock functions are enabled.");
-            }
-            enable_clock
-        } else {
-            return Err(format!("Expecting a boolean, but found {}", enable_clock).into());
-        }
+    let dump_stdout = if matches.is_present("dump-stdout") {
```

Comment:
Is this equivalent to `let dump_stdout = DEFAULT_DUMP_STDOUT || matches.is_present("dump-stdout");`? (Similarly for `dump_stderr`.) And I'm wondering if there's any point in having `DEFAULT_DUMP_STDOUT` in the code at all seeing as it will probably remain `false` for ever.